components/layout_2020/table/layout.rs

```diff
@@ -2823,16 +2815,22 @@ fn get_size_percentage_contribution_from_style(
         };
 
     LogicalVec2 {
-        inline: get_contribution_for_axis(size.inline, max_size.inline),
-        block: get_contribution_for_axis(size.block, max_size.block),
+        inline: get_contribution_for_axis(&size.inline, &max_size.inline),
+        block: get_contribution_for_axis(&size.block, &max_size.block),
     }
 }
 
-fn get_outer_sizes_from_style(
+fn get_outer_sizes_for_measure(
```

Comment:
This is getting the outer sizes for the process of measuring the table, correct? Maybe `get_outer_sizes_for_measurement` might be a bit clearer here. The former name is a bit ambiguous, because it could mean, "get the outer sizes taking a 'measure' as an argument."

```suggestion
fn get_outer_sizes_for_measure(
```